.travis.yml

```diff
@@ -6,5 +6,14 @@ jdk:
 after_success:
   - ./gradlew jacocoTestReport coveralls
 
+before_cache:
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+
+sudo: false
```

Comment:
Doesn't matter, it's running on the new infrastructure since it's a new repo anyway.